[2024-12-27T16:26:18.951+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-27T16:26:18.969+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: xgboost_drift_pipeline.test_prediction_after_retrain manual__2024-12-27T16:26:06.610699+00:00 [queued]>
[2024-12-27T16:26:18.978+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: xgboost_drift_pipeline.test_prediction_after_retrain manual__2024-12-27T16:26:06.610699+00:00 [queued]>
[2024-12-27T16:26:18.980+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-27T16:26:18.997+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): test_prediction_after_retrain> on 2024-12-27 16:26:06.610699+00:00
[2024-12-27T16:26:19.005+0000] {standard_task_runner.py:72} INFO - Started process 1935 to run task
[2024-12-27T16:26:19.008+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'xgboost_drift_pipeline', 'test_prediction_after_retrain', 'manual__2024-12-27T16:26:06.610699+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/train_xgboost_pipeline.py', '--cfg-path', '/tmp/tmp7ofoxn0a']
[2024-12-27T16:26:19.012+0000] {standard_task_runner.py:105} INFO - Job 30: Subtask test_prediction_after_retrain
[2024-12-27T16:26:19.029+0000] {logging_mixin.py:190} WARNING - /usr/local/lib/python3.10/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-27T16:26:19.065+0000] {task_command.py:467} INFO - Running <TaskInstance: xgboost_drift_pipeline.test_prediction_after_retrain manual__2024-12-27T16:26:06.610699+00:00 [running]> on host 42dc6dafdabb
[2024-12-27T16:26:19.165+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='xgboost_drift_pipeline' AIRFLOW_CTX_TASK_ID='test_prediction_after_retrain' AIRFLOW_CTX_EXECUTION_DATE='2024-12-27T16:26:06.610699+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-27T16:26:06.610699+00:00'
[2024-12-27T16:26:19.168+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-27T16:26:19.189+0000] {train_xgboost_pipeline.py:765} INFO - Testing prediction on current data...
[2024-12-27T16:26:19.274+0000] {train_xgboost_pipeline.py:665} INFO - Downloaded data/Ingestion/Financial_with_Sentiment.csv from MinIO to /tmp/Financial_with_Sentiment.csv
[2024-12-27T16:26:19.293+0000] {train_xgboost_pipeline.py:665} INFO - Downloaded data/Ingestion/retrained_model.pkl from MinIO to /tmp/retrained_model.pkl
[2024-12-27T16:26:19.674+0000] {logging_mixin.py:190} WARNING - /app/dags/train_xgboost_pipeline.py:778 DtypeWarning: Columns (2161) have mixed types. Specify dtype option on import or set low_memory=False.
[2024-12-27T16:26:19.790+0000] {logging_mixin.py:190} WARNING - /app/dags/train_xgboost_pipeline.py:685 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2024-12-27T16:26:21.288+0000] {logging_mixin.py:190} WARNING - /app/dags/train_xgboost_pipeline.py:685 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2024-12-27T16:26:21.306+0000] {train_xgboost_pipeline.py:791} INFO - Test Prediction Evaluation - MSE: 57.32, R2: 0.72
[2024-12-27T16:26:21.307+0000] {logging_mixin.py:190} INFO - Test Prediction Metrics:
MSE: 57.32
R2: 0.72
[2024-12-27T16:26:21.309+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-27T16:26:21.321+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-27T16:26:21.322+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=xgboost_drift_pipeline, task_id=test_prediction_after_retrain, run_id=manual__2024-12-27T16:26:06.610699+00:00, execution_date=20241227T162606, start_date=20241227T162618, end_date=20241227T162621
[2024-12-27T16:26:21.354+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-27T16:26:21.373+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-27T16:26:21.376+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
